pkg/instance/execprog.go

```diff
@@ -70,7 +74,7 @@ func SetupExecProg(vmInst *vm.Instance, mgrCfg *mgrconfig.Config, reporter *repo
 	}
 	if opt != nil {
 		ret.OptionalConfig = *opt
-		if ret.StraceBin != "" {
+		if !mgrCfg.StraceBinOnTarget && ret.StraceBin != "" {
```

Comment:
Shouldn't we set `ret.StraceBin = "strace"` when `mgrCfg.StraceBinOnTarget`? I assume it's in PATH in this configuration.

Reply:
I don't mind using posix_spawnp.